expo.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+)
+
+// Expo ...
+type Expo struct {
+	Version string
+	Workdir string
+}
+
+// installExpoCLI runs the install npm command to install the expo-cli
+func (e Expo) installExpoCLI() error {
+	args := []string{"install", "-g"}
+	if e.Version != "latest" {
+		args = append(args, "expo-cli@"+e.Version)
+	} else {
+		args = append(args, "expo-cli")
+	}
+
+	cmd := command.New("npm", args...)
+	cmd.SetStdout(os.Stdout)
+	cmd.SetStderr(os.Stderr)
+
+	log.Donef("$ " + cmd.PrintableCommandArgs())
```

Comment:
NOTE: It is better to use log.Donef("$ %s", cmd.PrintableCommandArgs()), in case there are special characters that are not quoted (like %)

Reply:
Wow, nice catch 👍 .